protostar/testing/cheatcodes/example_cheatcode.py

```diff
@@ -0,0 +1,40 @@
+from typing import Any
+
+from typing_extensions import Protocol
+
+from protostar.starknet import Cheatcode
+from protostar.testing.fuzzing.strategy_descriptor import StrategyDescriptor
+from protostar.testing.test_config import TestConfig, TestMode
+
+
+class ExampleCallable(Protocol):
+    def __call__(self, **kwargs: StrategyDescriptor):
+        ...
+
+
+class ExampleCheatcode(Cheatcode):
+    def __init__(
+        self,
+        syscall_dependencies: Cheatcode.SyscallDependencies,
+        test_config: TestConfig,
+    ):
+        super().__init__(syscall_dependencies)
+        self.test_config = test_config
+
+    @property
+    def name(self) -> str:
+        return "example"
+
+    def build(self) -> ExampleCallable:
+        return self.example
+
+    def example(self, *args: Any, **kwargs: Any) -> None:
+        self.test_config.convert_mode_to(TestMode.FUZZ)
+
+        assert bool(args) != bool(
+            kwargs
+        ), "you have to provide either args or kwargs to the example cheatcode"
```

Comment:
[Functionality] `AssertionError` tells the user to report the issue. Raising `CheatcodeException` would be more user-friendly.

Reply:
`given` only accepts `kwargs`. `example` should do that too.

Reply:
[use cheatcode exception instead of assert](https://github.com/software-mansion/protostar/pull/947/commits/49a8c53ee21304110aa266c771336b06b33d55a4)

Reply:
@karol-bisztyga please take a look at how `given` is implemented and replicate its behaviour here. `given` does not take any `args` for simplicity